surfsense_backend/app/utils/connector_service.py

```
@@ -959,3 +959,96 @@ async def search_linkup(self, user_query: str, user_id: str, mode: str = "standa
+                
+                # Create a more descriptive title for Discord messages
+                title = f"Discord: {channel_name}"
+                if message_date:
+                    title += f" ({message_date})"
+                    
+                # Create a more descriptive description for Discord messages
+                description = chunk.get('content', '')[:100]
+                if len(description) == 100:
+                    description += "..."
+                    
+                # For URL, we can use a placeholder or construct a URL to the Discord channel if available
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)